alf/examples/actor_critic.py

```diff
@@ -132,9 +135,20 @@ def create_algorithm(env,
             env.observation_spec(), fc_layer_params=value_fc_layers)
 
     encoding_net = None
-    if encoding_fc_layers:
-        encoding_net = EncodingNetwork(
-            env.observation_spec(), fc_layer_params=encoding_fc_layers)
+    if encoding_fc_layers or encoding_conv_layers:
+        if len(encoding_fc_layers) > 1:
+            # support different activation function for hidden and output layers
+            encoding_net = AlfEncodingNetwork(
+                input_tensor_spec=env.observation_spec(),
+                conv_layer_params=encoding_conv_layers,
+                preprocessing_combiner=None,
```

Comment:
Do you have to specify `preprocessing_combiner` here? The default should work. And it is specified here. The user won't be able to change it through gin config

Reply:
it just encode  a single observation here , so `preprocessing_combiner=None`

Reply:
Making this change will cause the the model use a different activation function, when using AlfEncodingNetwork, the activation is None. But now it will use relu. Did you test whether you can still train successfully?